helm/alfresco-process-infrastructure/values.yaml

```diff
@@ -316,6 +322,23 @@ alfresco-deployment-service:
       value: "{{ .Values.connectorVolume.permission }}"
     - name: APPLICATIONS_DATABASE_EXTERNAL
       value: "{{ .Values.applications.database.external }}"
+    {{- if .Values.applications.connectors.emailConnector.username }}
```

Comment:
unless you added for a specific reason, the `if` would not be necessary.

Reply:
see if that's consistent with other "" empty values in the past

Reply:
the reason for adding condition is to avoid adding empty variables to pod, if users choose not to set at helm deploy than there will be empty variables. If an empty variable is needed for some reason I can remove the condition.

Reply:
i think it makes sense yes, make sure that is consistent with other env variables set in the past. Resolving this comment